src/CMakeLists.txt

```diff
@@ -20,6 +20,8 @@ set(${TARGET_NAME}_PRIVATE_HEADERS
     logging_p.h
 )
 
+add_definitions(-DQTMQTT_LIBRARY)
```

Comment:
I prefer that this is called QTMQTT_BUILD_LIBRARY as it better expresses that the library is being built